runtime/strings/bytes.cpp

```diff
@@ -69,9 +71,10 @@ SortInt hook_BYTES_bytes2int(
   return move_int(result);
 }
 
-unsigned long get_ui(mpz_t i) {
+unsigned long get_ui_named(mpz_t i, std::string caller) {
```

Comment:
We don't need to copy this argument:
```suggestion
unsigned long get_ui_named(mpz_t i, std::string const& caller) {
```

You could (if you were so inclined) even make the argument type `char const*` and avoid constructing a `std::string` from the literals being passed in, but the const-ref is probably fine here.